js/ui/button_group.js

```diff
@@ -0,0 +1,230 @@
+import $ from "../core/renderer";
+import Widget from "./widget/ui.widget";
+import Button from "./button";
+import CollectionWidget from "./collection/ui.collection_widget.edit";
+import registerComponent from "../core/component_registrator";
+import { extend } from "../core/utils/extend";
+import BindableTemplate from "./widget/bindable_template";
+import { isPlainObject } from "../core/utils/type";
+
+const BUTTON_GROUP_CLASS = "dx-buttongroup",
+    BUTTON_GROUP_WRAPPER_CLASS = "dx-buttongroup-wrapper",
+    BUTTON_GROUP_ITEM_CLASS = "dx-buttongroup-item";
+
+var ButtonCollection = CollectionWidget.inherit({
+    _renderItemContent(options) {
+        options.container = $(options.container).parent();
+        this.callBase(options);
+    },
+
+    _focusTarget() {
+        return this.$element().parent();
+    },
+
+    _keyboardEventBindingTarget() {
+        return this._focusTarget();
+    },
+
+    _refreshContent() {
+        this._prepareContent();
+        this._renderContent();
+    },
+
+    _itemClass() {
+        return BUTTON_GROUP_ITEM_CLASS;
+    }
+});
+
+/**
+ * @name dxButtonGroup
+ * @inherits Widget
+ * @module ui/button_group
+ * @export default
+ */
+const ButtonGroup = Widget.inherit({
+    _getDefaultOptions() {
+        return extend(this.callBase(), {
+            /**
+             * @name dxButtonGroupOptions.buttonType
+             * @type Enums.ButtonType
+             * @default 'normal'
+             */
+            buttonType: "normal",
+
+            /**
+             * @name dxButtonGroupOptions.hoverStateEnabled
+             * @type boolean
+             * @default false
+             */
+            hoverStateEnabled: true,
```

Comment:
Choose something one
             * @default false
            hoverStateEnabled: true,

Reply:
It is my mistake :)